Fix stale lint-skill-manifest path in create-skill guide + ADR 0008

## Context

During PR #12912 (peer-naming skill authoring, 2026-06-11), following the `create-skill` authoring guide verbatim threw `Error: Cannot find module '.../ai/scripts/lint-skill-manifest.mjs'` (MODULE_NOT_FOUND). The documented lint command points at a path that no longer exists.

## The Problem

The skill-manifest lint script lives at `ai/scripts/lint/lint-skill-manifest.mjs` (note the `lint/` subdir), but several live, agent-consumed docs still reference the pre-`lint/` path `ai/scripts/lint-skill-manifest.mjs`. Any agent running the documented command verbatim hits MODULE_NOT_FOUND and must re-derive the real path. The CI workflow itself already uses the correct path (so CI is unaffected — the drift is docs-only), which is exactly why it went unnoticed.

## The Architectural Reality

Live (non-archive) stale references, verified via `grep -rn 'ai/scripts/lint-skill-manifest'`:

- `.agents/skills/create-skill/references/skill-authoring-guide.md:183` — the Verification-checklist command (the one that bit #12912).
- `learn/agentos/decisions/0008-skill-anatomy-and-authoring-contract.md:77, 125, 145` — ADR 0008, the canonical skill-anatomy decision substrate (3 refs).
- `test/playwright/unit/ai/scripts/lint/lintSkillManifest.spec.mjs:22` — a `test.describe(...)` label (cosmetic; the test file's own path already carries `lint/`).

Real path confirmed: `ls ai/scripts/lint/lint-skill-manifest.mjs` → exists.

## The Fix

Replace `ai/scripts/lint-skill-manifest.mjs` → `ai/scripts/lint/lint-skill-manifest.mjs` (and the bare describe-label) across the 5 live references. One coherent cleanup, one PR (not micro-PRs).

## Decision Record impact

`aligned-with ADR 0008` — a factual path correction within the ADR's prose; no change to the decision itself.

## Acceptance Criteria

- [ ] `skill-authoring-guide.md:183` lint command uses `ai/scripts/lint/lint-skill-manifest.mjs`.
- [ ] ADR 0008's 3 references corrected to the `lint/` path.
- [ ] The test `describe` label aligned (cosmetic).
- [ ] `grep -rn 'ai/scripts/lint-skill-manifest'` over non-archive paths returns zero.
- [ ] `node ai/scripts/lint/lint-skill-manifest.mjs --base origin/dev` runs (proves the corrected command works).
- [ ] `resources/content/**` archive snapshots left untouched.

## Out of Scope

- The ~55 `resources/content/{discussions,pulls}/**` archive references (frozen historical record — they document what was written at the time).
- Any change to the lint script itself or the CI workflow (both already correct).

## Related

- Empirical anchor: PR #12912 (peer-naming skill authoring). Surfaced as a friction→gold spawned-task during that PR.

Release classification: boardless — substrate-doc hygiene, non-blocking.

Origin Session ID: 0bc22bf2-9ded-4cdb-b351-182c3ab1d16f

Retrieval Hint: "stale lint-skill-manifest path create-skill guide ADR 0008 MODULE_NOT_FOUND"

--- a/.agents/skills/create-skill/references/skill-authoring-guide.md
+++ b/.agents/skills/create-skill/references/skill-authoring-guide.md
@@ -180,7 +180,7 @@ Before pushing your new skill, check:
 - [ ] Does the `SKILL.md` body provide the explicit project-relative path to the reference file?
 - [ ] Is `.agents/skills/skills.manifest.json` updated to mirror the frontmatter and governance fields?
 - [ ] Is there a corresponding symlink for the new skill in `.claude/skills/`?
-- [ ] Does `node ai/scripts/lint-skill-manifest.mjs --base origin/dev` pass locally?
+- [ ] Does `node ai/scripts/lint/lint-skill-manifest.mjs --base origin/dev` pass locally?
 
 ## PR-Open Gates for Skill Changes (create OR modify)
 

--- a/learn/agentos/decisions/0008-skill-anatomy-and-authoring-contract.md
+++ b/learn/agentos/decisions/0008-skill-anatomy-and-authoring-contract.md
@@ -74,7 +74,7 @@ Every skill MUST have a corresponding entry in `.agents/skills/skills.manifest.j
 - `claudeSymlinkRequired` (boolean; mandatory `.claude/skills/<name>` symlink discipline)
 - `downstreamDocsTargets` (array of docs files that must be touched when this skill changes)
 
-Schema enforced by `.agents/skills/skills.manifest.schema.json`; lint enforced by `ai/scripts/lint-skill-manifest.mjs` at PR-merge time (extended by PR #11438 with `oversizedWorkflowMaps` + `maxPositiveDeltaBytes` for recursive Map-vs-Atlas enforcement).
+Schema enforced by `.agents/skills/skills.manifest.schema.json`; lint enforced by `ai/scripts/lint/lint-skill-manifest.mjs` at PR-merge time (extended by PR #11438 with `oversizedWorkflowMaps` + `maxPositiveDeltaBytes` for recursive Map-vs-Atlas enforcement).
 
 ### 2.5 Claude Symlink Mandate
 
@@ -122,7 +122,7 @@ The skill-anatomy contract codified by this ADR is consumed by:
 
 ### 3.3 Mechanical-enforcement consumers
 
-- **`ai/scripts/lint-skill-manifest.mjs`** — manifest contract validation at PR-merge time (per §2.4); PR #11438 extends with recursive Map-vs-Atlas enforcement (`oversizedWorkflowMaps` + `maxPositiveDeltaBytes`)
+- **`ai/scripts/lint/lint-skill-manifest.mjs`** — manifest contract validation at PR-merge time (per §2.4); PR #11438 extends with recursive Map-vs-Atlas enforcement (`oversizedWorkflowMaps` + `maxPositiveDeltaBytes`)
 - **`.agents/skills/skills.manifest.schema.json`** — JSON schema for manifest contract; PR #11424 removes `triggers` from required fields
 
 ---
@@ -142,7 +142,7 @@ The skill-anatomy contract spans the following substrate locations. Future-amend
 
 - **`.agents/skills/skills.manifest.json`** — per-skill manifest entries per §2.4
 - **`.agents/skills/skills.manifest.schema.json`** — JSON schema validating the manifest
-- **`ai/scripts/lint-skill-manifest.mjs`** — runtime enforcement of the schema + per-skill budgets
+- **`ai/scripts/lint/lint-skill-manifest.mjs`** — runtime enforcement of the schema + per-skill budgets
 
 ### 4.3 Cross-substrate references (META-prose authority)
 

--- a/test/playwright/unit/ai/scripts/lint/lintSkillManifest.spec.mjs
+++ b/test/playwright/unit/ai/scripts/lint/lintSkillManifest.spec.mjs
@@ -19,7 +19,7 @@ import {
     validateManifestSchema
 } from '../../../../../../ai/scripts/lint/lint-skill-manifest.mjs';
 
-test.describe('ai/scripts/lint-skill-manifest (#11275)', () => {
+test.describe('ai/scripts/lint/lint-skill-manifest (#11275)', () => {
     const scriptPath = path.resolve(process.cwd(), 'ai/scripts/lint/lint-skill-manifest.mjs');
 
     test('CLI passes against the repository manifest', () => {

